gulpfile.js

```diff
@@ -81,7 +81,7 @@ gulp.task('styles', () => {
     var themeFile = `./theme/${themeName()}/style.scss`;
     beginPipe('./static/styles/**/*.{css,sass,scss}')
         .pipe(sassGrapher.ancestors())
-        .pipe(header(fs.readFileSync(themeFile, 'utf8')))
+        .pipe(header(fs.readFileSync(themeFile, 'utf8')+"// don't remove me! \n")) // READ: https://github.com/schul-cloud/schulcloud-client/pull/588
```

Comment:
![image](https://user-images.githubusercontent.com/9433996/36721266-b102806c-1baa-11e8-821a-85e2c5abb39b.png)

The `homework.scss` still make problems on my machine..